src/rewrite/vct/rewrite/DeserializeOriginToLLVMOrigin.scala

```diff
@@ -0,0 +1,17 @@
+package vct.col.rewrite
+
+import vct.col.ast.Deserialize
+import vct.col.origin.{LLVMOrigin, Origin}
+
+case object DeserializeOriginToLLVMOrigin extends RewriterBuilder {
+  override def key: String = "DeserializeOriginToLLVMOrigin"
```

Comment:
shorter and lowercase, maybe `llvmOrigin`?

Reply:
I corrected the style and naming issues. Regarding the aforementioned type check issue: Perhaps it would be possible to make `ColLLVMParser` invoke the `DeserializeLLVMOrigin` rewriter? It would make sure the origins are converted before any error could be raised.